dataline-integrations/src/main/java/io/dataline/integrations/Integrations.java

```diff
@@ -42,7 +42,10 @@
       new IntegrationMapping("dataline/integration-singer-postgres-destination:0.1.1")),
   BIGQUERY_TARGET(
       UUID.fromString("e28a1a10-214a-4051-8cf4-79b6f88719cd"),
-      new IntegrationMapping("dataline/integration-singer-bigquery-destination:0.1.1"));
+      new IntegrationMapping("dataline/integration-singer-bigquery-destination:0.1.1")),
+  CSV_TARGET(
+      UUID.fromString("8442ee76-cc1d-419a-bd8b-859a090366d4"),
+      new IntegrationMapping("dataline/integration-singer-csv-destination:dev"));
```

Comment:
Should this be `0.1.1` instead of dev?

Reply:
yeah! will switch before merging.

Reply:
i might even remember to push the integration version first this time. 😅